charts/kuberpult/Earthfile

```diff
@@ -46,7 +44,7 @@ chart.yaml:
     WORKDIR /kp
 
     COPY Chart.yaml.tpl .
-    
+
```

Comment:
nitpick: A linter/formatter for earthfiles would be great

Reply:
There are no available formatters for Earthfiles currently. I configured my editor to treat Earthfiles as Dockerfiles and it formats them as such.